Caltech-IPAC/firefly
# Firefly-1510: Color shows in table tabs

#### [Firefly-1510](https://jira.ipac.caltech.edu/browse/FIREFLY-1510) / ([design ticket : Firefly-1345](https://jira.ipac.caltech.edu/browse/FIREFLY-1345)): Color shows in table tabs

  - Table use color swatch
  - TAP includes button to set table title
  - Small TAP panel clean up
  
#### Testing
  - https://fireflydev.ipac.caltech.edu/firefly-1510-color/firefly
  - load multiple catalog tables
    - Color feedback in tabs
      - You should see colors on each tab
      - click color the change the color
      - bring up drawing layer, change color of a layer, the the tab will change
    - Search Title
       - From TAP dialog click `Custom Search Title` button, set new title
       - Experiment with the dialog after two or three searches
 - Also test from Euclid: https://firefly-1510-color.irsakudev.ipac.caltech.edu/applications/euclid
